src/views/HomeView.vue

```diff
@@ -268,12 +280,22 @@ const updateCurrentlyPlayingQueueWidgetRow = function () {
           )
         ) {
           widgetRows.value.queue.players.push(player);
+          widgetRows.value.queue.players.sort((a, b) =>
+            a.name.localeCompare(b.name),
```

Comment:
```suggestion
            a.display_name.localeCompare(b.display_name),
```

Reply:
always use display_name as that takes into account any custom name the user gave to a player

Reply:
Will do